docs/csharp/local-functions-vs-lambdas.md

```diff
@@ -81,8 +81,7 @@ assigned at the `return` statement.
 The analysis that enables that analysis enables the fourth difference.
 Depending on their use, local functions can avoid heap allocations that
 are always necessary for lambda expressions. If a local function is never
-converted to a delegate and none of the variables captured by a locatl function
-are converted to delegates, the compiler can avoid heap allocations. 
+converted to a delegate and one of the variables captured by the local function are captured by other lambdas or local functions that are converted to delegates, the compiler can avoid heap allocations. 
```

Comment:
is this one of the variables, or none of the variables, as in the original version. One makes little sense.
There should be a comma after "converted to a delegate"
are captured --> is captured